Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -353,6 +356,296 @@ void MolDraw2D::drawMolecule(const ROMol &mol, const std::string &legend,
   }
 }
 
+namespace {
+void get2DCoordsMol(RWMol &mol, double &offset, double spacing, double &maxY,
+                    double &minY, int confId, bool shiftAgents,
+                    double coordScale) {
+  try {
+    MolOps::sanitizeMol(mol);
+  } catch (const MolSanitizeException &e) {
+    mol.updatePropertyCache(false);
+  }
+  const bool canonOrient = true;
+  RDDepict::compute2DCoords(mol, NULL, canonOrient);
+  MolDraw2DUtils::prepareMolForDrawing(mol);
```

Comment:
sanitizeMol has already kekulized (or already failed) so turn of kekulization here.  Otherwise the error gets propagated.